docs/content/docs/data-structures/overview.md

```diff
@@ -56,7 +56,7 @@ Key-value data structures are the most common choice for many scenarios.
 
 ### Common issues and best practices for key-value DDSes
 
-- Storing a counter in a map will have unexpected behavior. Use the SharedCounter instead.
+- Storing a counter in a map will have unexpected behavior. Use the [SharedCounter]({{< relref "counter.md" >}}) instead.
```

Comment:
But currently, that article doesn't actually explain the unexpected behavior. Either the explanation should be there or it should be here. Either way a concrete example would be good.

Reply:
Added to the Counter article.